.kokoro/docker/Dockerfile

```diff
@@ -30,32 +30,19 @@ RUN apt update \
     libxml2-dev \
     libxslt1-dev \
     openssl \
-    portaudio19-dev \
-    python-pyaudio \
     zlib1g-dev \
- && apt clean
+ && apt-get clean
 
-###################### Install python 3.7.12
-
-# Download python 3.7.12
-RUN wget https://www.python.org/ftp/python/3.7.12/Python-3.7.12.tgz
-
-# Extract files
-RUN tar -xvf Python-3.7.12.tgz
-
-# Install python 3.7.12
-RUN ./Python-3.7.12/configure --enable-optimizations
-RUN make altinstall
 
 ###################### Check python version
 
-RUN python --version
-RUN which python
+RUN python3 --version
+RUN which python3
 
 # Setup Cloud SDK
-ENV CLOUD_SDK_VERSION 369.0.0
+ENV CLOUD_SDK_VERSION 489.0.0
 # Use system python for cloud sdk.
-ENV CLOUDSDK_PYTHON /usr/bin/python
+ENV CLOUDSDK_PYTHON python3.12
```

Comment:
Should we use something like `ENV CLOUDSDK_PYTHON $(which python3)`? (Not sure if that's valid Docker)